### ISSUE TITLE
Exporting the network matrix using updated hnf v2

### ISSUE THREAD

**Issue by msdogan** [ucd-cws/calvin-network-tools#45]
@jrmerz I updated my calvin-network-data and calvin-network-tools repos. I tried to export the matrix in two ways: nodejs and npm, but it gave me error. Did I do something wrong?

nodejs:
```
AD3+msdogan@CWS-Amargosa  /z/calvin-network-tools (master)
$ node --max-old-space-size=4096 bin/cnf.bin.js matrix --verbose --format=tsv --start=1921-10 --stop=2003-10 --ts=. --fs=:tab: --to=network --outnodes=nodes --max-ub=1000000000
module.js:341
    throw err;
    ^

Error: Cannot find module 'commander'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (Z:\calvin-network-tools\nodejs\index.js:3:15)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```
npm:

```
AD3+msdogan@CWS-Amargosa  /z/calvin-network-tools (master)
$ cnf matrix --verbose --format=tsv --start=1921-10 --stop=2003-10 --ts=. --fs=:tab: --to=network --outnodes=nodes --max-ub=1000000000
*** CNF Arguments ***
  _allowUnknownOption: false
  _name: matrix
  _noHelp: false
  _description: Create a network matrix
  format: tsv
  header: true
  start: 1921-10
  stop: 2003-10
  ts: .
  fs: :tab:
  to: network
  outnodes: nodes
  maxUb: 1000000000
  data: z:\calvin-network-data\data
  runtime: Z:\nodejs\node_modules\calvin-network-tools\HEC_Runtime
  verbose: true
[]
**********************

Running matrix command.

Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\crawler\toGeojson.js:60
    var origin = nodePathLookup[originPath].data;
                                           ^

TypeError: Cannot read property 'data' of undefined
    at Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\crawler\toGeojson.js:60:44
    at Array.forEach (native)
    at module.exports.networkFiles.data.region.forEach.networkFiles.data.region.forEach.networkFiles.data.node.forEach.networkFiles.data.link.forEach.networkFiles.data.node.forEach.networkFiles.data.region.forEach (Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\crawler\toGeojson.js:54:26)
    at Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\crawler\index.js:16:7
    at onResp (Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\git.js:40:11)
    at Z:\nodejs\node_modules\calvin-network-tools\node_modules\hobbes-network-format\lib\git.js:51:9
    at ChildProcess.exithandler (child_process.js:204:7)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
```

**Comment by jrmerz** [comment]
run
```
npm install
```
from the root directory

**Comment by msdogan** [comment]
is it `npm install` or `npm install -g calvin-network-tools`?

**Comment by jrmerz** [comment]
this command
```
node --max-old-space-size=4096 bin/cnf.bin.js
```
is running locally from the repo.

to run from the global command line install, use
```
cnf matrix --verbose --format=tsv --start=1921-10 --stop=2003-10 --ts=. --fs=:tab: --to=network --outnodes=nodes --max-ub=1000000000
```

**Comment by msdogan** [comment]
@jrmerz there is no rush for this. I run the command as an admin and got the following.


<img width="487" alt="untitled" src="https://cloud.githubusercontent.com/assets/14085662/21075335/b90ca228-bec3-11e6-9d02-82b5c707d0fd.png">

**Comment by jrmerz** [comment]
@msdogan k, push fixes at calvin-network-tools@v2.0.2

There was a bug with using relative paths as well as bug with windows.  Let me know if you are able to run.

**Comment by msdogan** [comment]
@jrmerz yes, it worked! I was able to export the network matrix.
